packages/create-cloudflare/templates/hello-world-durable-object/ts/src/index.ts

```diff
@@ -14,7 +14,7 @@ import { DurableObject } from "cloudflare:workers";
  */
 
 /** A Durable Object's behavior is defined in an exported Javascript class */
-export class MyDurableObject extends DurableObject {
+export class MyDurableObject extends DurableObject<Env> {
```

Comment:
also added this fix here

Reply:
Doh! I see now. This is imported from the `cloudflare:workers` module, which is what brings in this class.